android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt

```diff
@@ -172,6 +179,30 @@ class MainActivity : DaggerAppCompatActivity() {
         }
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        binding.toolbar.children.forEach {
+            when (it) {
+                is ActionMenuView -> {
+                    it.children.filterIsInstance<ActionMenuItemView>().forEach { menuItemView ->
+                        setRippleColor(menuItemView, binding.isIndigoBackground)
+                    }
+                }
+                is AppCompatImageButton -> setRippleColor(it, binding.isIndigoBackground)
+            }
+        }
+        return super.onCreateOptionsMenu(menu)
+    }
+
+    private fun setRippleColor(view: View, isIndigoBackground: Boolean) {
+        (view.background as? RippleDrawable)?.setColor(
+            ColorStateList.valueOf(this.getThemeColor(
+                if (isIndigoBackground) {
+                    R.attr.colorSurface
```

Comment:
If we use DarkTheme, colorSurface is invisible when isIndigoBackground is true.
colorPrimarySurface looks better.

Reply:
Can you use `colorPrimarySurface`??

Reply:
I think colorPrimarySurface cannot be seen because it is the same color as the background.
![screenshot](https://user-images.githubusercontent.com/74615/73610216-57895980-4618-11ea-9103-fd3e9aa2b698.png)

I think using white color is fine.
![white](https://user-images.githubusercontent.com/74615/73610253-a7682080-4618-11ea-8678-958c4b00f27c.png)

DarkTheme
![white2](https://user-images.githubusercontent.com/74615/73610267-c070d180-4618-11ea-9843-770115c101a2.png)

What are your thoughts on that?

Reply:
The color of icons seems to changed from `colorPrimarySurface` to `colorOnPrimary` on base code.
https://github.com/DroidKaigi/conference-app-2020/pull/452/files#diff-14153ff2fbf8b01642f55ee0af1ccdc7L219-L225

And the definition of `colorOnPrimary` is white on both light and dark theme.
https://github.com/DroidKaigi/conference-app-2020/search?q=colorOnPrimary&unscoped_q=colorOnPrimary

So I agree, Its good to use white color for ripple. It's same as icon color.
But I prefer to use semantic color definition (likes R.attr.colorOnPrimary etc) instead of direct color definition (Color.WHITE) to control colors by theme.

Reply:
Use colorOnPrimary for ripple color.

70dcb376f15c2ceb54064b2df512830a5f479b13